### PR TITLE
Signup component created

### PR DESCRIPTION
**Pull request by Ninad99**
@all-contributors please add @Ninad99 for code

### CODE REVIEW

**Comment by ArnasDickus** [comment]
Hello until you fix travis test I can't review your project.

Travis problem:
Line 54:30:  'setIsPasswordHidden' is assigned a value but never used

File: src/pages/Register/RegisterPage/Register.module.scss [hunk 1]
```diff
@@ -1,353 +1,504 @@
...
-            width: 100%;
-
-            .signup {
-                .headlines {
-                    justify-content: center;
-
-                    .headline {
-                        font-size: 18px;
-                    }
+// .signupcomponent {
```
Comment (ArnasDickus) [inline src/pages/Register/RegisterPage/Register.module.scss]: Why did you left this long chunk of code?

File: src/pages/Register/RegisterPage/RegisterPage.jsx [hunk 2]
```diff
@@ -109,15 +107,81 @@ function RegisterPage({ setName, setEmail, setPassword, onFormSubmit, span }) {
...
                         </button>
                     </div>
+                    <h2>Already a member?&nbsp;&nbsp;<Link to="/login">Sign In</Link></h2>
                     {span && (
                         <Span className={classes.errorLabel} content={span} />
                     )}
                 </form>
             </div>
         </div>
+        // <div className={classes.signupcomponent}>
```
Comment (ArnasDickus) [inline src/pages/Register/RegisterPage/RegisterPage.jsx]: Why did you leave a huge chunk of code?

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello thanks for adding PR. Here's my concerns:

1) Sign up wrong font size. The design has 36px,  your 20px. Also text is too low.
2) Name, email address, password. Text is bold as well as 14px not 16px.
3) Avoid using h1, h3, buttons as a selectors use classes instead.

I suggest looking once again to [figma](https://www.figma.com/file/KN4yf9KRGRLxPBlUy4VrixZS/Visual-Music?node-id=1605%3A1955). Click on text look at font-size try to make it look closer to design.

Also I'm aware about navbar issue. I will respond regarding that issue a little later.

File: src/pages/Register/RegisterPage/Register.module.scss (hunk 1 above)
Comment (Ninad99) [reply src/pages/Register/RegisterPage/Register.module.scss]: Sorry. Thought you'd like to have a look at the previous styles once. Removing them!

File: src/pages/Register/RegisterPage/RegisterPage.jsx (hunk 2 above)
Comment (Ninad99) [reply src/pages/Register/RegisterPage/RegisterPage.jsx]: Removing

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello looks much better. Now, all is left to do is. 
Improve error message according to [design](https://www.figma.com/file/KN4yf9KRGRLxPBlUy4VrixZS/Visual-Music?node-id=1605%3A1978).

So you can improve components/units/Span/Span element. Or create your own error element based on that span, and put it similiar to design. To avoid messing up with other designs don't delete span element from components/units/Span/Span.

**Comment by Ninad99** [comment]
Hey! I modified the styles on the existing Span element so that it matches the design. Please have a look!

**Review by ArnasDickus — APPROVED** [review]
Looks great. Merging.

**Pull request merged**